pkg/controllers/machinesetsync/machineset_sync_controller.go

```diff
@@ -86,7 +93,8 @@ func (r *MachineSetSyncReconciler) SetupWithManager(mgr ctrl.Manager) error {
 			handler.EnqueueRequestsFromMapFunc(util.ResolveCAPIMachineSetFromObject(r.MAPINamespace)),
 			builder.WithPredicates(util.FilterNamespace(r.CAPINamespace)),
 		).
-		Complete(r); err != nil {
+		Complete(r)
+	if err != nil {
```

Comment:
Curious why this change, seems a backwards step in terms of clean code to me

Reply:
Oh snap I forgot to revert this. Was playing with less nesting when refactoring other bits. Will change back.